Dockerfile.bundle

```diff
@@ -0,0 +1,32 @@
+FROM scratch
```

Comment:
Maybe, reuse the same `bundle.Dockerfile` file instead?
How about modifying that file for the ditto contents.

Reply:
Most repositories had Dockerfile.<variant>, hence changed the renamed the file. If `bundle.Dockerfile` is preferred I can revert the filename .